README.md

```diff
@@ -17,4 +17,8 @@ Kapitola 4 - radost z mergovania
 s mergovanim ti iba poradim updatnut si project skor ako zacnes robit 
 beriem na vedomie 
 
-kapitola 5 ZAVER
\ No newline at end of file
+kapitola 5 - pull requesty
+Vidis zmysel PR (pull requestov) ?
+
+kapitola 6 - naozajstny zaver
+Ako sa ti to paci?
```

Comment:
zatial najlepsie, ako to vidis ty dobre pocuvam ?

Reply:
myslim si ze ano... este budu databazy a to bude este vacsie vzruso

Reply:
ale to uz vidim na zajtra